runtime/src/core/runner.js

```diff
@@ -58,6 +124,33 @@ export class GrainRunner {
     }
   }
 
+  // [HACK] Temporarily used while we transition to AS-based runtime
+  _makeGrainString(v) {
+    let buf = this.encoder.encode(v);
+    let userPtr = this.managedMemory.malloc((4 * 2) + (((v.length - 1) / 4) + 1));
+    let ptr = userPtr / 4;
+    let view = this.managedMemory._view;
+    view[ptr] = GRAIN_STRING_HEAP_TAG;
+    view[ptr + 1] = v.length;
+    let byteView = this.managedMemory.u8view;
+    for (let i = 0; i < buf.length; ++i) {
+      byteView[i + (ptr * 4) + 8] = buf[i];
+    }
+    return userPtr | GRAIN_GENERIC_HEAP_TAG_TYPE;
+  }
+
+  // [HACK] Temporarily used while we transition to AS-based runtime
+  grainToString(v) {
```

Comment:
Can we call this something like `grainStringToJSString`?

Reply:
I'm fine changing the name, but I don't think that's a good pick. It takes an arbitrary grain value, calls Grain's internal `toString`, and returns the JS string version of it. Maybe `grainValueToString`?

Reply:
Sounds good. Clearly I didn't know what it was doing 😛